### DISCUSSION TITLE
Achievements: Historical data

### DISCUSSION THREAD

**Discussion by milosmns** [community/community#19431] [Other Feature Feedback, Questions, & Ideas]
It looks like this feature is not really counting historical data... is it?
If no, how do we look through the profile's history to get achievements? 🙂  (I'd be happy with a manual trigger to do this)

Update: I did search through the other discussions, and didn't find this... I was expecting to find it though.

**Comment by ahmedjsofficial** [comment]
yes! its should be

**Reply by milosmns** [reply]
Hm ok 😀 
I am not sure really as I don't have a great overview of my stats, but the numbers seem quite low tbh.. ok let's wait a bit more then and check again

**Reply by milosmns** [reply]
Hm actually maybe it's not counting private repos, is that right? For example the pull request thingy is at 3 but it should be like 5k or something

**Comment by melvincwng** [comment]
@milosmns, as far as I'm aware, some of your historical data is not taken into account when determining whether you've hit the requirements to get a specific achievement.

Hope this clarifies.
